modules/utxo-bin/README.md

```diff
@@ -9,6 +9,14 @@ formatting them for terminal output.
 npm install --global @bitgo/utxo-bin
 ```
 
+# Upgrade
+```
+cd modules/utxo-bin
+yarn build
+```
+
```

Comment:
I don't think we specific instructions for this module - the toplevel README should over it

doesn't `yarn install` on the toplevel work as well?

Reply:
No it doesn't

Reply:
Does `yarn build` on the toplevel do the right thing?

Reply:
> error Command "build" not found.

Reply:
The `yarn install` (or just `yarn`) command on the toplevel should build all packages. If it doesn't work that after that we have a bug that we need to fix separately.